setup.py

```diff
@@ -23,6 +23,7 @@
         'scipy',
         'click',
         'tables',
+        'loguru',
```

Comment:
@collijk should I also put loguru in the requirements.txt? Is that even used?

Reply:
It is not.  We can get rid of it.